pygmt/datasets/samples.py

```diff
@@ -215,36 +215,18 @@ def load_sample_bathymetry(**kwargs):
     return data
 
 
-def load_usgs_quakes(**kwargs):
+def _load_usgs_quakes():
     """
-    (Deprecated) Load a table of global earthquakes from the USGS as a
-    pandas.DataFrame.
-
-    .. warning:: Deprecated since v0.6.0. This function has been replaced with
-       ``load_sample_data(name="usgs_quakes")`` and will be removed in
-       v0.9.0.
+    Load a table of global earthquakes from the USGS as a pandas.DataFrame.
 
     This is the ``@usgs_quakes_22.txt`` dataset used in the GMT tutorials.
```

Comment:
Please see the comments at https://github.com/GenericMappingTools/pygmt/pull/2301#discussion_r1062048779.
```suggestion
```